gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -0,0 +1,152 @@
+              into horizontal cards by replacing this style with
+              {' '}<code className="nx-code">.nx-card-container--column</code> below.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-container--column</code></td>
+            <td className="nx-cell">Modifier of <code className="nx-code">.nx-card-container</code></td>
+            <td className="nx-cell">
+              Container for horizontally orientated cards. Horizontal cards can be transformed into vertical cards by
+              replacing this style with <code className="nx-code">.nx-card-container--row</code> above.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Basic card element.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-header</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Card header element, typically wraps an <code className="nx-code">&lt;h3&gt;</code>.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__call-out</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Wraps the card call out (if any). The call out typically consists of a number, sparkline, icon, or
+              other small graphic. It should not be free form text.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-call__call-out-text</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              In the case where the call out consists of text (e.g. 99%) it should be wrapped in this class (typically
+              in a <code className="nx-code">&lt;span&gt;</code>).
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__text</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">Free-form text. Should be concise.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-footer</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">Card footer. Typically provides a link to more information.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card--vertical-alt</code></td>
```

Comment:
I notice that this class seems to be safe to use on horizontal cards - it has no effect on them.  Is that a deliberate fact that we want to support or is it just coincidence? I think it would be more convenient for users if it was supported.

Reply:
> I notice that this class seems to be safe to use on horizontal cards - it has no effect on them. Is that a deliberate fact that we want to support or is it just coincidence? I think it would be more convenient for users if it was supported.

It doesn't effect horizontal cards because it only has an effect when the card is a child of `.nx-card-container--row`.